ios/patches/PromisesObjC.podspec

```diff
@@ -0,0 +1,22 @@
+Pod::Spec.new do |s|
```

Comment:
is this patch folder really needed ?

Reply:
It's needed because google didn’t merge my [PR](https://github.com/google/promises/pull/224) yet - I will remove it in future